puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_hammer_plugin.yaml

```diff
@@ -0,0 +1,26 @@
+---
+- job-template:
+    name: test_hammer_plugin_{name}_{branch}
+    project-type: pipeline
+    sandbox: true
+    parameters:
+    - string:
+        name: hammer_plugin_url
+        description: 'git url of the hammer plugin repo'
+    - string:
+        name: hammer_plugin_branch
+        default: master
+        description: 'git branch of the hammer plugin to base on'
+
+    dsl:
+      !include-raw:
+      - pipelines/lib/rvm.groovy
+      - pipelines/lib/git.groovy
+      - pipelines/test/hammer_plugin_template_test.groovy
+
+- project:
+    name: hammer_cli_foreman_kubevirt
```

Comment:
Let's use dashes please, as that's our typical convention and the repository name

Reply:
@ehelms , are you sure? the gem is with an underscore,